contracts/peripheral/IncurDebt.sol

```diff
@@ -383,23 +367,27 @@ contract IncurDebt is OlympusAccessControlledV2, IIncurDebt {
         ITreasury(treasury).repayDebtWithOHM(ohmToRepay);
     }
 
-    /// @notice withdraws sOHM  to _to address
+    /// @notice withdraws gOHM
     /// - msg.sender must be a borrower
     /// - _amount (in OHM) must be less than or equal to depositedOhm - debt
-    /// @param _amount amount of sOHM to withdraw
-    /// @param _to address to send _amount
-    function withdraw(uint256 _amount, address _to) external override isBorrower(msg.sender) {
+    /// @param _amount amount of gOHM to withdraw
+    function withdraw(uint256 _amount) external override isBorrower(msg.sender) {
         if (_amount == 0) revert IncurDebt_InvaildNumber(_amount);
         Borrower storage borrower = borrowers[msg.sender];
 
-        updateCollateralInSOHM();
         if (_amount > getAvailableToBorrow()) revert IncurDebt_AmountAboveBorrowerBalance(_amount);
```

Comment:
Amount is gohm, getAvailableToBorrow() returns borrowers balance in sohm.

Reply:
ahh yes